nservicebus/upgrades/5to6/raven-upgrade.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Upgrade NServiceBus.RavenDB from 3 to 4
+summary: Instructions on how to upgrade NServiceBus.RavenDB from NServiceBus Versions 5 to 6
+tags:
+ - upgrade
+ - migration
+ - raven
+related:
+- nservicebus/upgrades/5to6
+---
+
+## Namespace change
+
+The `RavenDBPersistence` has moved from `NServiceBus.Persistence` namespace to `NServiceBus`. There is no need to have an extra using statement if you have the NuGet package installed. 
+
+## Saga correlation properties
+
+In NServiceBus.RavenDB, more than one correlation property was never allowed. In NServiceBus 6, this has become the default for all persistence implementations.
```

Comment:
I was thinking about that as well. It seemed like a nice thing to know, but maybe it doesn't matter. I'll remove it.